requirements/requirements.txt

```diff
@@ -1,6 +1,6 @@
 holidays>=0.9
 matplotlib~=3.0
-numpy~=1.16
+numpy~=1.21
```

Comment:
We probably want to split this change into another PR.

Reply:
Is this the reason why `test_empirical_distribution` fails? If yes, totally.

Reply:
No, I don't think these are related.